uncontrast_studies/processors/distribution_of_effects_across_parameters.py

```diff
@@ -0,0 +1,89 @@
+import itertools
+
+from django.contrib.postgres.expressions import ArraySubquery
+from django.db.models import QuerySet, OuterRef, F, Count, Func
+from django.db.models.functions import JSONObject
+from uncontrast_studies.models import UnConExperiment, UnConSuppressedStimulus, UnConsciousnessMeasure, UnConSample
+from uncontrast_studies.processors.base import BaseProcessor
+
+
+class DistributionOfEffectsAcrossParametersGraphDataProcessor(BaseProcessor):
+    """ """
+
+    def __init__(self, experiments: QuerySet[UnConExperiment], **kwargs):
+        super().__init__(experiments=experiments, **kwargs)
+        breakdown = kwargs.pop("continuous_breakdown")
+        self.breakdown = breakdown[0]
+
+    def process(self):
+        process_func = getattr(self, f"process_{self.breakdown}")
+        return process_func()
+
+    def process_suppressed_stimuli_duration(self):
+        subquery = UnConSuppressedStimulus.objects.filter(experiment__significance=OuterRef("series_name")).annotate(
+            value=F("duration")
+        )
+        return self._aggregate_query_by_breakdown(subquery)
+
+    def process_number_of_stimuli(self):
+        subquery = UnConSuppressedStimulus.objects.filter(experiment__significance=OuterRef("series_name")).annotate(
+            value=F("number_of_stimuli")
+        )
+        return self._aggregate_query_by_breakdown(subquery)
+
+    def process_unconsciousness_measure_number_of_trials(self):
+        subquery = UnConsciousnessMeasure.objects.filter(experiment__significance=OuterRef("series_name")).annotate(
+            value=F("number_of_trials")
+        )
+        return self._aggregate_query_by_breakdown(subquery)
+
+    def process_unconsciousness_measure_number_of_participants_in_awareness_test(self):
+        subquery = UnConsciousnessMeasure.objects.filter(experiment__significance=OuterRef("series_name")).annotate(
+            value=F("number_of_participants_in_awareness_test")
+        )
+        return self._aggregate_query_by_breakdown(subquery)
+
+    def process_sample_size_included(self):
+        subquery = UnConSample.objects.filter(experiment__significance=OuterRef("series_name")).annotate(
+            value=F("size_included")
```

Comment:
probably should be `sample_size_included`

Reply:
should be the "name of the field" on the UnConSample object - which I think is without the prefix

Reply:
in this case, change the naming of `sample_size_excluded` in the next processor

Reply:
true , would fix